oak-core/src/test/java/org/apache/jackrabbit/oak/plugins/index/property/MultiPropertyOrTest.java

```diff
@@ -121,22 +121,17 @@ public void unionSortResultCount() throws Exception {
         root.commit();
 
         List<Integer> nodes = Lists.newArrayList();
-        Random r = new Random();
-        int seed = -2;
+        Random r = new Random(1);
```

Comment:
Why? You want the test to be deterministic?

Reply:
Yes.